webapp/src/main/webapp/src/app/pages/tests/create/test-create.page.ts

```diff
@@ -24,9 +24,10 @@ import { Notification, NotificationEvent, NotificationService, NotificationType
 import { ServicesService } from '../../../services/services.service';
 import { TestsService } from '../../../services/tests.service';
 import { SecretsService } from '../../../services/secrets.service';
-import { Operation, Service } from '../../../models/service.model';
+import {Contract, Operation, Service} from '../../../models/service.model';
```

Comment:
Could you please follow the same format? => `{ Contract, Operation, Service }`

Reply:
Made this change.